proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -954,3 +957,27 @@ message DeepHealthCheckRequest {
 message DeepHealthCheckResponse {
     temporal.server.api.enums.v1.HealthState state = 1;
 }
+
+
+message UpdateActivityOptionsRequest {
+    // Namespace of the workflow which scheduled this activity
+    string namespace_id = 1;
+    // Id of the workflow which scheduled this activity
+    string workflow_id = 2;
+    // Run Id of the workflow which scheduled this activity
+    // if empty - latest workflow is used
+    string run_id = 3;
+    // Id of the activity we're updating
+    string activity_id = 4;
+
+    // Activity options. Partial updates are accepted and controlled by update_mask
+    temporal.api.activity.v1.ActivityOptions activity_options = 6;
+
+    // Controls which fields from `activity_options` will be applied
+    google.protobuf.FieldMask update_mask = 7;
```

Comment:
this should just embed the workflowservice request as a message

Reply:
I try that at first, and it this triggers 2 different linter errors.
All other request are using namespace_id, but request to front end is using namespace name.
plus few other small differences.
I prefer to keep the separated.

Reply:
Right.. look at how all the others in this file are done, e.g. RespondActivityTaskCompletedRequest or ResetWorkflowExecutionRequest: you need one field with namespace_id, and then another field for the forwarded request (where the namespace field is ignored in favor of the namespace_id at the top level of this one)

Reply:
Uh, this was merged without this comment addressed. Can you please fix this to work like all the other RPCs in this file?